app/views/books/show.html.erb

```diff
@@ -1,5 +1,26 @@
 <h1><%= @book.title %></h1>
+  <link rel="stylesheet" href="styles/5star.css">
+
 <p>Genre: <%= @book.genre.capitalize %></p>
 <p>Language: <%= @book.language.capitalize %></p>
+<br>
+<h3>Leave a review</h3>
+<%= simple_form_for [@book, @review] do |f| %>
+<!--   <%= f.input :rating %> -->
+  <p>Rating *</p>
+  <%= f.collection_radio_buttons :rating, [[1], [2], [3], [4], [5]], :first, :last, item_wrapper_tag: false, boolean_style: :inline do |b| %>
+  <%= b.radio_button + b.label {''} %>
+  <% end %>
+  <%= f.input :review_content %>
+  <%= f.submit "Submit", class: "btn btn-primary" %>
+<% end %>
+<br>
+<h3>Reviews</h3>
+<div>
+  <% @book.reviews.each do |review| %>
+    <p><%= "⭐️" * review.rating %></p>
```

Comment:
In here, I think it would be good to add something like  <p>Review by: <%= review.user.username %></p>

Reply:
Perhaps also a review.date_created and perhaps an edit for the user who wrote it? Option to delete? Not sure how much of the CRUD we need here?

Reply:
I've added the username and the "created at" date to the reviews list. It is not beautified yet (separate card). The created_at date is showing with the time and timezone as well- is this okay? If not, do you know how to change it?

I also wrote in my above comment that I'd add edit/destroy later; I have a separate card for that in Trello bc I'm not sure how that will work with Pundit and everything. If you think I should add those methods now, I can.